docs/ref/settings.txt

```diff
@@ -385,6 +385,11 @@ where ``reason`` is a short message (intended for developers or logging, not for
 end users) indicating the reason the request was rejected.  See
 :doc:`/ref/csrf`.
 
+The default implementation ``django.views.csrf.csrf_failure`` takes an additional
+keyword parameter ``template_name`` that defaults to ``'csrf_fail.html'``. If a
```

Comment:
I prefer including `403` because I think the most important parallel is not with the setting name or view name (since after your PR, most developers will have no reason to even use the setting or reference the view directly). The most important parallel is with the other templates you are likely to have in the top-level of your templates directory, since that's something you are likely to look at daily, and it would be nice to have all your error-status templates look similar. (And perhaps you're right that `403_csrf.html` is best, so they'd also be grouped together in a directory listing.)

But in the end this is a bikeshed, and as the PR author you get to paint it. If you are not convinced by my argument above, feel free to keep it as-is :-)